### PR TITLE
fix rubocops

### PR DESCRIPTION
**Pull request by kbrock**

### CODE REVIEW

**Comment by kbrock** [comment]
update:

- rebase

File: lib/floe/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Floe
-  VERSION = "0.9.0".freeze
+  VERSION = "0.9.0"
```
Comment (Fryguy) [inline lib/floe/version.rb:4]: This one surprised me - I thought rubocop enforce freezing all constants 😕

File: lib/floe/workflow/reference_path.rb [hunk 2]
```diff
@@ -19,11 +19,11 @@ def initialize(*)
         super
 
         raise Floe::InvalidWorkflowError, "Invalid Reference Path" if payload.match?(/@|,|:|\?/)
+
         @path = JsonPath.new(payload)
                         .path[1..]
                         .map { |v| v.match(/\[(?<name>.+)\]/)["name"] }
-                        .map { |v| v[0] == "'" ? v.delete("'") : v.to_i }
-                        .compact
+                        .filter_map { |v| v[0] == "'" ? v.delete("'") : v.to_i }
```
Comment (Fryguy) [inline lib/floe/workflow/reference_path.rb:26]: Nice! I didn't know filter_map was available on the versions of Ruby we supported (presuming test cases exercise this line)

File: lib/floe/version.rb (hunk 1 above)
Comment (Fryguy) [reply lib/floe/version.rb:4]: Oh duh, there's a frozen string literal at the top (I never liked those, and almost think that's the actual fix we should do here).

File: lib/floe/workflow/reference_path.rb (hunk 2 above)
Comment (kbrock) [reply lib/floe/workflow/reference_path.rb:26]: just learned this one. `rubocop` does teach us good stuff. It is ruby>=2.7

**Review by Fryguy — APPROVED** [review]

**Pull request merged**